NitroxClient/MonoBehaviours/Gui/MainMenu/MainMenuMultiplayerPanel.cs

```diff
@@ -127,6 +147,25 @@ public void ShowAddServerWindow()
             shouldFocus = true;
         }
 
+        private void StartServer()
+        {
+            string path = Path.Combine(Environment.CurrentDirectory, "SubServer\\NitroxServer.exe");
```

Comment:
Looks like this won't work without installing Nitrox through the installer. Dev enviroment doesn't have "SubServer" dir.